### PR TITLE
update uptime.com RC notification scripts and sample

### PR DESCRIPTION
**Pull request by ShellyXueHan**
uptime.com has changed their alert payload. Here is the updated version of Rocket.Chat webhook script to use!

### CODE REVIEW

File: uptime/monitor-setup/uptime-rc-webhook-2023-11-16.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      statusText = `UP`;
+    }
+
+    let attachmentText = '';
+    let titleText = '';
+    let titleLink = '';
+    if (isUp) {
+      attachmentText += 'Back to normal now!';
+      titleText = 'More info';
+      titleLink = 'https://uptime.com';
```
Comment (WadeBarnes) [inline uptime/monitor-setup/uptime-rc-webhook-2023-11-16.js:51]: `titleLink` is not being used.

File: uptime/monitor-setup/uptime-rc-webhook-2023-11-16.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    if (isUp) {
+      attachmentColor = `#36A64F`;
+      statusText = `UP`;
+    }
+
+    let attachmentText = '';
+    let titleText = '';
+    let titleLink = '';
+    if (isUp) {
+      attachmentText += 'Back to normal now!';
```
Comment (WadeBarnes) [inline uptime/monitor-setup/uptime-rc-webhook-2023-11-16.js]: Missing an indication of how long the monitored service was down for; see previous RC script.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Looking good.  Just a couple of comments.

**Review by WadeBarnes — APPROVED** [review]
LGTM

**Review by w8896699 — APPROVED** [review]
Looks good to me

**Pull request merged**